src/pybind/cephfs/cephfs.pyx

```diff
@@ -238,9 +239,9 @@ cdef make_ex(ret, msg):
     """
     ret = abs(ret)
     if ret in errno_to_exception:
-        return errno_to_exception[ret](msg)
+        return errno_to_exception[ret](ret, msg)
```

Comment:
I'm not convinced many callers will handle it specially - our own ceph cli just prints the exception for example. Callers won't always catch these, so I think we should be optimizing for the user, rather than the developer here.

We could make Error inherit from EnvironmentError instead, which will use your errno and strerror attributes:

```
>>> class Error(EnvironmentError):
...     pass
... 
>>> print(Error(22, 'bad input foo'))
[Errno 22] bad input foo
```